glusterd2/xlator/utils.go

```diff
@@ -35,14 +35,14 @@ func (e OptionNotFoundError) Error() string {
 // key should be in the [<graph>].<xlator>.<name> form.
 // Returns an error otherwise.
 func FindOption(k string) (*options.Option, error) {
-	// Interested only in <xlator>.<name> part of the key as optMap is indexed
+	// Interested only in <xlator>.<name> part of the key as OptMap is indexed
```

Comment:
This function was provided to specifically not export `optMap` outside the package.

Reply:
done